examples/cartesian_reconstruction.py

```diff
@@ -0,0 +1,98 @@
+"""
+Neuroimaging cartesian reconstruction
+=====================================
+
+Author: Chaithya G R
+
+In this tutorial we will reconstruct an MRI image from the sparse kspace
+measurments.
+
+Import neuroimaging data
+------------------------
+
+We use the toy datasets available in pysap, more specifically a 2D brain slice
+and the acquisition cartesian scheme.
+"""
+
+# Package import
+from modopt.math.metrics import ssim
+from mri.numerics.reconstruct import sparse_rec_fista
```

Comment:
try to sort the import alphabetically (`reconstruct` after `fourier`)

Reply:
Hmm, I dont think I did that detailed sorting. Will get this done